### DISCUSSION TITLE
Next release wish list items to simplify aircraft setup.

### DISCUSSION THREAD

**Discussion by SimitarKnut** [EdgeTX/edgetx#1870] [Ideas]
I would like to see on the model set up, the choice between electric or fuel, motor/engine. Then on the mix page, instead of doing this otherwise, have the option to assign a switch to mix with the throttle to:

If electric, override the throttle to cut off.

If fuel, if throttle is below a certain position to cut the throttle, but if above a certain position, but below half, (or a programmable weight range) to hold the throttle instead of cutting it.

Thanks to all you good people who are doing your best to make our hobby better!

Claude
AKA Simitar Knut

**Comment by SimitarKnut** [comment]
I just set up my first new aircraft on Edge 2.6.0

I started with the usual first four channels but when I wanted to add  channel 6 for flaps and 7 for light controls I was perplexed that these channels weren't responding.

I found I had to add those channels on the 'Mix' page before they would output on the receiver/servo.

I think that if it has to be on the 'Mix' page, it should be added automatically when a channel is assigned on the 'Input' Page.

Claude
AKA Simitar Knut

**Reply by pfeerick** [reply]
Good point... that would be a nice addition, if the corresponding mixer line is empty. 

Having said that, you can instead skip the inputs page, and use the Mixes page instead. i.e. skip the inputs page unless you want to be able to do more complex things like having switchable dual rates, expo. i.e. where the input to the mixer would vary.

**Comment by Eldenroot** [comment]
Better to open a new issue
